### PR TITLE
Set spec file to build fat RPMs

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Set spec file to build far RPMs

Part of inmanta/irt#438

# Self Check:

- [ ] Attached issue to pull request
- [ ] Changelog entry

### CODE REVIEW

File: inmanta.spec [hunk 1]
```diff
@@ -14,67 +18,185 @@
...
-# Use the correct python for bycompiling
-%define __python %{_p3}
+%description -n inmanta-oss-server
 
-%description
+%description -n inmanta-oss-agent
 
 %prep
 %setup -q -n inmanta-%{sourceversion_egg}
+%setup -T -D -a 1 -n inmanta-%{sourceversion_egg}
```
Comment (bartv) [inline inmanta.spec:83]: why are these three lines?

Comment (arnaudsjs) [reply inmanta.spec:83]: They unpack `Source1`, `Source2` and `Source3` respectively

Comment (bartv) [reply inmanta.spec:83]: right, -n is the target dir.

**Review by bartv — APPROVED** [review]

File: inmanta.spec (hunk 1 above)
Comment (arnaudsjs) [reply inmanta.spec:83]: Yes

**Review by sanderr — APPROVED** [review]

**Comment by arnaudsjs** [comment]
@sanderr I requested a re-review because I fixed a small issue. See last commit. The packages on GitHub have a different layout then the package I created manually.

**Review by sanderr — APPROVED** [review]

**Comment by arnaudsjs** [comment]
I am requesting a re-review because some additional changes happened.

**Review by sanderr — APPROVED** [review]

**Review by bartv — APPROVED** [review]

**Pull request merged**